### PR TITLE
Edm 120 part 1

### PR DESCRIPTION
**Pull request by nicholashibbits**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

- This PR includes the static UI for the new licenses and certifications search page in the GI Comparison Tool.
- I'm part of the Education Data Migration team, however VEBT has ownership of Comparison Tool, where these components are located.
- The original plan was to use a flipper to hide the work in progress, however I chose to disable the route housing the new UI entirely, so it can't be reached at all. There will likely be 1-2 more PRs adding more static UI components, at which point I'll match the page within a Route component so the new page can be navigated to.

## Related issue(s)

- EDM-120 begin the frontend build of WEAMS Public in Comparison Tool

## Testing done

- No testing required for static UI updates

## Screenshots

There are no actual updates to the UI being merged here. These are child components that will be displayed upon completing development of feature, but for now exist only in the code. If needed, I'm happy to reenable the route in my dev environment and provide screenshots.

## What areas of the site does it impact?

GI Comparison Tool

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/gi/containers/LicenseCertificationSearch.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import LicenseCertificationSearchFields from '../components/LicenseCertificationSearchFields';
+
+function LicenseCertificationSearch() {
+  return (
+    <div>
+      <section className="lc-wrapper vads-u-padding-x--2p5 small-screen:vads-u-padding-x--2">
```
Comment (GovCIOLiz) [inline src/applications/gi/containers/LicenseCertificationSearch.jsx:7]: Platform is updating breakpoint prefixes - all but medium-screen are being renamed. In this case `small-screen` should be replaced with `mobile-lg` (More info in the canvas in platform-design-system on slack)

File: src/applications/gi/sass/gi.scss [hunk 2]
```diff
@@ -271,3 +271,12 @@
     cursor: pointer;
   }
 }
+
+.lc-wrapper {
```
Comment (GovCIOLiz) [inline src/applications/gi/sass/gi.scss:275]: might be able to use `vads-u-display--flex vads-u-flex-direction--column` instead

File: src/applications/gi/containers/LicenseCertificationSearch.jsx (hunk 1 above)
Comment (nicholashibbits1) [reply src/applications/gi/containers/LicenseCertificationSearch.jsx]: Thanks for the tip! The suggested changes have been made.

File: src/applications/gi/sass/gi.scss (hunk 2 above)
Comment (nicholashibbits1) [reply src/applications/gi/sass/gi.scss]: You're right. The change has been made

**Review by GovCIOLiz — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**